### ISSUE TITLE
Define product vision

### ISSUE THREAD

**Issue by HarryPotia** [WeAreSnook/connected-camden#26]

**Comment by mathewtrivett** [comment]
@vandagomes shall we create a bit of a template to articulate the Product vision here? I think it would be useful to have something we refer back to. I'm going to put together a bit of a suggestion here.

**Comment by vandagomes** [comment]
Yes @mathewtrivett I was thinking the same! Do we have a template for this or should I make from the scratch?

**Comment by mathewtrivett** [comment]
I put this together on Miro as a potential starting point? Feel free to pick it up and tweak it. Maybe it could be something we do with Camden too?
https://miro.com/app/board/o9J_lfjTCPo=/
